--- apertium-eu-es-0.3.3/Makefile.am.orig	2015-12-28 21:40:37.775569141 +0100
+++ apertium-eu-es-0.3.3/Makefile.am	2015-12-28 21:47:46.115551127 +0100
@@ -178,7 +178,7 @@
 	rm -rf modes
 	mv modes.bak modes
 	test -d $(DESTDIR)$(apertium_modesdir) || mkdir $(DESTDIR)$(apertium_modesdir)
-	$(INSTALL_DATA) $(PREFIX1).mode $(apertium_modesdir)
+	$(INSTALL_DATA) $(PREFIX1).mode $(DESTDIR)$(apertium_modesdir)
 	rm -f $(PREFIX1).mode
 
 CLEANFILES = $(TARGETS)
